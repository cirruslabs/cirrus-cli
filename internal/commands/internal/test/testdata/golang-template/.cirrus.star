load("github.com/cirrus-templates/golang@a4e91ca453a4ade8f41013fca0888536d680f51d", "detect_tasks")

def main(ctx):
    return [detect_tasks()]
