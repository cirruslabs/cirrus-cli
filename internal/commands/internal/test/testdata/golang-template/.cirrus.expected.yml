task:
  - name: Tests
    container:
      image: golang:latest
      cpu: 2
      memory: 4096
    get_script:
      - go get ./...
    build_script:
      - go build ./...
    test_script:
      - go test ./...
